spec/services/case_create_service_spec.rb

```diff
@@ -25,189 +25,206 @@
-    end
-  end
+      it 'sets the result to "assign_responder"' do
+        ccs.call
+        expect(ccs.result).to eq :assign_responder
+      end
 
-  context 'ICO case' do
-    let(:received)          { 0.business_days.ago }
-    let(:deadline)          { 28.business_days.after(received) }
-    let(:internal_deadline) { 10.business_days.before(deadline) }
-    let(:foi)               { create :closed_case }
-    let(:params) do
-      ActionController::Parameters.new(
-      { 'case_ico' => {
-          'type'                    => 'Case::ICO::FOI',
-          'original_case_type'      => 'FOI',
-          'original_case_id'        => foi.id,
-          'ico_officer_name'        => 'Ian C. Oldman',
-          'ico_reference_number'    => 'ABC1344422',
-          'received_date_dd'        => received.day.to_s,
-          'received_date_mm'        => received.month.to_s,
-          'received_date_yyyy'      => received.year.to_s,
-          'external_deadline_dd'    => deadline.day.to_s,
-          'external_deadline_mm'    => deadline.month.to_s,
-          'external_deadline_yyyy'  => deadline.year.to_s,
-          'internal_deadline_dd'    => internal_deadline.day.to_s,
-          'internal_deadline_mm'    => internal_deadline.month.to_s,
-          'internal_deadline_yyyy'  => internal_deadline.year.to_s,
-          'message'                 => 'AAAAA'
-        }
-      })
+      it 'returns true' do
+        expect(ccs.call).to eq true
+      end
     end
-    let(:ccs) { CaseCreateService.new(manager, 'ico', params) }
-    let(:created_ico_case) { Case::ICO::FOI.last }
 
-    it 'uses the params provided' do
-      ccs.call
+    context 'ICO case' do
+      let(:received)          { 0.business_days.ago }
+      let(:deadline)          { 28.business_days.after(received) }
+      let(:internal_deadline) { 10.business_days.before(deadline) }
+      let(:foi)               { create :closed_case }
+      let(:params) do
+        ActionController::Parameters.new(
+          { 'case_ico' => {
+              'type'                    => 'Case::ICO::FOI',
+              'original_case_type'      => 'FOI',
+              'original_case_id'        => foi.id,
+              'ico_officer_name'        => 'Ian C. Oldman',
+              'ico_reference_number'    => 'ABC1344422',
+              'received_date_dd'        => received.day.to_s,
+              'received_date_mm'        => received.month.to_s,
+              'received_date_yyyy'      => received.year.to_s,
+              'external_deadline_dd'    => deadline.day.to_s,
+              'external_deadline_mm'    => deadline.month.to_s,
+              'external_deadline_yyyy'  => deadline.year.to_s,
+              'internal_deadline_dd'    => internal_deadline.day.to_s,
+              'internal_deadline_mm'    => internal_deadline.month.to_s,
+              'internal_deadline_yyyy'  => internal_deadline.year.to_s,
+              'message'                 => 'AAAAA'
```

Comment:
I find this derivative

Reply:
That's just the sound I make as I read your comment. With a few more exclamation marks, though.